gcloud-java-dns/src/main/java/com/google/gcloud/dns/Dns.java

```diff
@@ -260,14 +234,15 @@ public static RecordSetListOption type(RecordSet.Type type) {
      * specified. {@link ZoneField} provides a list of fields that can be used.
      */
     public static ZoneOption fields(ZoneField... fields) {
-      return new ZoneOption(DnsRpc.Option.FIELDS, ZoneField.selector(fields));
+      return new ZoneOption(DnsRpc.Option.FIELDS,
+          Helper.selector(ZoneField.REQUIRED_FIELDS, fields));
     }
   }
 
   /**
    * Class for specifying zone listing options.
    */
-  class ZoneListOption extends AbstractOption implements Serializable {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All methods are static.  Consider using a utility class instead. Alternatively, you could add a private constructor or make the class abstract to silence this warning.](https://www.codacy.com/app/mziccard/gcloud-java/file/2331164897/issues/source?bid=3123103&fileBranchId=3204660#l245)